doc/nrf/releases/release-notes-changelog.rst

```diff
@@ -286,44 +314,42 @@ Libraries
 
 This section provides detailed lists of changes by :ref:`library <libraries>`.
 
-Libraries for NFC
------------------
-
-* :ref:`nfc_ndef_parser_readme`
-
-  * :c:func:`nfc_ndef_msg_parse`: fixed declaration, added an assertion to avoid a potential usage fault and added a note in the function documentation.
-  * Renamed the :c:macro:`NFC_NDEF_PARSER_REQUIRED_MEM` macro.
-
 Bluetooth libraries and services
 --------------------------------
 
 * Added:
 
   * :ref:`ams_client_readme` library.
 
-* :ref:`gatt_dm_readme` library:
+* Updated:
```

Comment:
Remove this line. The idea is to have section for library type (Bluetooth library and services) and bullet point for each library changed, with changes listed under this library bullet point. This way, we mirror section/page hierarchy of the docset.

Reply:
This is of course global change.